packages/react-docs/src/pages/styles/icons.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Content from '../../components/content';
+import { Title } from '@patternfly/react-core';
+import { iconMap } from '@patternfly/react-icons/icons';
+import { css, StyleSheet } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Grid/styles.css';
+import {
+  global_spacer_md as spacerMd,
+  global_FontSize_sm as labelFontSize
+} from '@patternfly/react-tokens';
+
+const allIcons = Array.from(iconMap.entries());
+
+const extraStyles = StyleSheet.create({
+  iconCell: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: spacerMd.var
+  },
+  label: {
+    textAlign: 'center',
+    fontSize: labelFontSize.var
+  }
+});
+
+function Icons() {
+  return (
+    <Content>
+      <Title size="xxxl" withMargins>
+        Icons
+      </Title>
+      <div className={css(styles.grid)}>
+        {allIcons.map(([id, Icon]) => (
+          <div
+            key={id}
+            className={css(
+              styles.gridItem,
```

Comment:
Once the Gallery PR is merged a PR for Grid will be opened.  After that is merged this can be converted to use it.